components/Greeting/Greeting.tsx

```diff
@@ -0,0 +1,21 @@
+import styles from "../Greeting/Greeting.module.css";
+export type Name = {
+  name: string;
+};
+
+function Button({ name }: Name) {
+  return (
+    <div className={styles.grettingContainer}>
+      <div className={styles.header}>
+        Hello,<span className={styles.name}>{name}</span>
+      </div>
+      <img
+        src="/material-settings.svg"
+        alt=""
```

Comment:
If this image is used to link to a settings page (or open a modal), it should have aria labels or at least an `alt` which describes the content.

Reply:
Another question is, does this belong in this component?